### DISCUSSION TITLE
misc/new/

### DISCUSSION THREAD

**Discussion by giscus** [X-six-floor-slip-liu-X/X-six-floor-slip-liu-X.github.io#2] [Announcements]
# misc/new/

这是一个中学 OIer 的博客，他会在这里分享一些算法和题解，喜欢可以进来贴贴。

https://x-six-floor-slip-liu-x.github.io/misc/new/

**Comment by X-six-floor-slip-liu-X** [comment]
现在评论功能终于弄好了，大家有问题可以直接问我。
